diff --git a/win32-vcpkg.x86/installed/x86-windows/include/mpg123.h.in b/win32-vcpkg.x86/installed/x86-windows/include/mpg123.h.in
index 40792a3..edf803d 100644
--- a/win32-vcpkg.x86/installed/x86-windows/include/mpg123.h.in
+++ b/win32-vcpkg.x86/installed/x86-windows/include/mpg123.h.in
@@ -45,7 +45,7 @@
 #if defined(_MSC_VER) && !defined(MPG123_DEF_SSIZE_T)
 #define MPG123_DEF_SSIZE_T
 #include <stddef.h>
-typedef ptrdiff_t ssize_t;
+//typedef ptrdiff_t ssize_t;
 #endif

 #ifndef MPG123_NO_CONFIGURE /* Enable use of this file without configure. */
